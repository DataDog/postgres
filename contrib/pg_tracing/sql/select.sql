CREATE EXTENSION pg_tracing;
SELECT pg_tracing_reset();


SELECT traces from pg_tracing_info;

/*dddbs='postgres.db',traceparent='00-00000000000000007ed39623a1d7b2d2-7ed39623a1d7b2d2-01'*/ SELECT 1;
SELECT trace_id, parent_id, resource from pg_tracing_spans;
