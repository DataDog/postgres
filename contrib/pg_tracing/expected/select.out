CREATE EXTENSION pg_tracing;
SELECT pg_tracing_reset();
 pg_tracing_reset 
------------------
 
(1 row)

SELECT traces from pg_tracing_info;
 traces 
--------
      0
(1 row)

/*dddbs='postgres.db',traceparent='00-00000000000000007ed39623a1d7b2d2-7ed39623a1d7b2d2-01'*/ SELECT 1;
 ?column? 
----------
        1
(1 row)

SELECT trace_id, parent_id, resource from pg_tracing_spans;
      trace_id       |      parent_id      |   resource    
---------------------+---------------------+---------------
 9138813148648157906 | 9138813148648157906 | ExecutorStart
(1 row)

